frontend/app/src/data/psl.ts

```diff
@@ -1,5 +1,5 @@
 // Public suffixes list
-// Source: https://github.com/lupomontero/psl/blob/main/data/rules.json
+// Source: https://github.com/lupomontero/psl/blob/main/data/rules.js
```

Comment:
```suggestion
// Source: https://github.com/lupomontero/psl/blob/c445ac9c9ebe7a795335e11b1d4831c1bed8dbb2/data/rules.json
```

Since you are fixing the link this is not the correct link. 
The correct one is the one in the suggestion.